src/components/activity/ActivityDescription.tsx

```diff
@@ -0,0 +1,36 @@
+import { BoldTypography, StyledTypography } from '@/app/detail/ui/styledComponent'
+import { ActivityType } from '@/types/interfaces'
+import { Typography } from '@mui/material'
+
+type ActivityDescription = {
+  [key in ActivityType]: string | ((...args: any[]) => React.ReactNode)
+}
+
+const activityDescription: ActivityDescription = {
+  [ActivityType.CREATE_TASK]: () => <StyledTypography> created task. </StyledTypography>,
+  [ActivityType.ASSIGN_TASK]: (to: string) => (
+    <>
+      <StyledTypography> assigned task to </StyledTypography>
+      <BoldTypography>{to}.</BoldTypography>
+    </>
+  ),
+  [ActivityType.WORKFLOWSTATE_UPDATE]: (from: string, to: string) => (
+    <>
+      <StyledTypography> changed status from </StyledTypography>
+      <BoldTypography>{from}</BoldTypography>
+      <StyledTypography> to </StyledTypography>
+      <BoldTypography>{to}.</BoldTypography>
+    </>
+  ),
+}
+
+type ActivityDescriptionProps = {
+  activityType: ActivityType
+  args?: any[]
+}
+
+export default function ActivityDescription({ activityType, args = [] }: ActivityDescriptionProps) {
+  const description = activityDescription[activityType]
+  const renderDescription = typeof description === 'function' ? description(...args) : description
+  return <> {renderDescription}</>
+}
```

Comment:
I don't think we need this as well. Including this, we are having 3 different modules just to handle static rendering of the activity logs. Instead of creating component to render different activity types (which will bring this on the client side rendering), why not create a simpler object that has the style definition and text definition. I think we can handle this in much simpler way with few lines of code.